segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/ThrottlerCalculator.java

```diff
@@ -50,13 +50,13 @@
      * Number of items in the Commit Backlog above which throttling will apply.
      */
     @VisibleForTesting
-    static final int COMMIT_BACKLOG_COUNT_THRESHOLD = 300;
+    static final int COMMIT_BACKLOG_COUNT_THRESHOLD = 100;
 
     /**
      * Amount of time (millis) to increase throttling by for each incremental increase of the Commit Queue, over the threshold.
      */
     @VisibleForTesting
-    static final int THROTTLING_MILLIS_PER_COMMIT_OVER_LIMIT = 4;
+    static final int THROTTLING_MILLIS_PER_COMMIT_OVER_LIMIT = 50;
```

Comment:
50ms of delay seems really aggressive. Can we get away with 20?

Reply:
If you look at this PR's description, do those sample throttling delays look too aggressive? I think that if we are able to write 500 writes to BK but not able to cache those 500 writes, waiting 20 seconds is acceptable given the dire situation we might be in.